client/src/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux'
+import { reducer as redux_form } from 'redux-form'
+import auth_reducer from './authReducer'
+
+const allReducers = combineReducers({
+  form: redux_form,
+  auth: auth_reducer
```

Comment:
Why mixing camelCasing and snakeCasing? I see both casings in project. Please stick with camelCasing, prefereably

Reply:
Fixed!